express/features/your-account.feature

```diff
@@ -25,6 +25,58 @@ Feature:
 
   Scenario: The user tries to change their phone number but needs a new SMS code
 
+    Given the user has logged in and arrived on the your account page
+    When they click on the link that points to "/change-phone-number"
+    Then they should be directed to the following page: "/change-phone-number"
+
+    Given that the user is on the "/change-phone-number" page
```

Comment:
I think we can drop this step - we've checked they're on the right page above and this step effectively types the URL into the address bar which will work but might end up masking a bug in future.

Reply:
Done.